src/containers/LoginFormContainer.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import FacebookLogin from "react-facebook-login";
+import { BASE_URL } from "../constants";
+import axios from "axios";
+
+export default class LoginFormContainer extends Component {
+  state = {
+    isLoggedIn: false,
+    userID: "",
+    firstName: "",
+    lastName: "",
+    email: "",
+    picture: "",
+    token: ""
+  };
+
+  componentClicked = () => console.log("clicked");
+
+  responseFacebook = response => {
+    console.log(response);
+
+    this.setState({
+      isLoggedIn: true,
+      userID: response.userID,
+      firstName: response.first_name,
+      lastName: response.last_name,
+      email: response.email,
+      picture: response.picture.data.url,
+      token: response.accessToken
+    });
+
+    console.log(response.accessToken);
+  };
+
+  addNewUser = () => {
+    axios
+      .post(`${BASE_URL}/login`, {
+        firstName: this.state.first_name,
+        lastName: this.state.last_name,
+        email: this.state.email,
+        picture: this.state.picture.data.url,
+        token: this.state.accessToken
+      })
+      .then(() => {
+        this.props.history.push("/");
+      })
+      .catch(err => console.log(err));
+  };
+
+  render() {
+    let fbCotent;
+
+    if (this.state.isLoggedIn) {
```

Comment:
you can use destructuring state, code will look better:
**const { isLoggedIn, picture, firstName} = this.state;**  below render()
and then you can use just isLoggedIn, without repeating this.state